### PR TITLE
Feat/test main.py

### PR DESCRIPTION
**Pull request by NicoShanstrom**
creates test file and test methods for service call. too late to past the PR template, can do that later if necessary.

### CODE REVIEW

File: tests/test_main.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    print("Request headers:", response.request.headers)
+    print("Request body:", response.request.content.decode('utf-8'))
+    print("Response status code:", response.status_code)
+    print("Response JSON:", response.json())
+
+    assert response.status_code == 200
+    data = response.json()
+    assert "advice" in data
+
+def test_root():
```
Comment (Steddy1Love) [inline tests/test_main.py:42]: I don't think this is necessary, but I appreciate the test for root. I left it in the main.py for check once deployed.

**Review by Steddy1Love — APPROVED** [review]
Incredible work! I feel like I can refer to this for applicaiton of pytest and python.

File: tests/test_main.py (hunk 1 above)
Comment (NicoShanstrom) [reply tests/test_main.py:42]: I agree. Just wasn’t sure what we wanted to do with that. Keeping it didn’t seem to hurt.

**Comment by NicoShanstrom** [comment]
It was a lot of looking at some docs and AI help. I need to dive into understanding each piece of it better. I wanted to get it up for us. But yes, we have something to work off of as at least a template and file tree approach for how it could look as an example. CircleCI isn’t passing bc I think the environment variable in circle isn’t there for OPENAI_API_KEY for the config.yml file. I think I have the commented out cors in there because I saw that on a doc and wanted to play with it later but didn’t want to lose my thought. Figured since we use that in rails backend to front end that we may need it at some point in the microservice.

**Pull request merged**